concepts/05 Widgets/Form/01 Getting Started with Form/50 Configure Item Labels.md

```diff
@@ -0,0 +1,187 @@
+                dataField: "notes",
+                colSpan: 2,
+                label: {
+                    alignment: "center"
+                }
+            }]
+        });
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-form
+        [formData]="employee"
+        [colCount]="2"
+        labelLocation="top"
+        [showColonAfterLabel]="false">
+        <dxi-item dataField="name"></dxi-item>
+        <dxi-item dataField="position"></dxi-item>
+        <dxi-item dataField="hireDate"></dxi-item>
+        <dxi-item dataField="officeNumber"></dxi-item>
+        <dxi-item 
+            dataField="notes" 
+            [colSpan]="2">
+            <dxo-label alignment="center">
+            </dxo-label>
+        </dxi-item>
+    </dx-form>
+
+    <!-- tab: app.component.ts -->
+    import { Component } from '@angular/core';
+
+    @Component({
+        selector: 'app-root',
+        templateUrl: './app.component.html',
+        styleUrls: ['./app.component.css']
+    })
+    export class AppComponent {
+        employee = {
+            name: 'John Heart',
+            position: 'CEO',
+            hireDate: new Date(2012, 4, 13),
+            officeNumber: 901,
+            notes: 'John has been in the Audio/Video industry since 1990.'
+        }
+    }
+
+    <!-- tab: app.module.ts -->
+    import { BrowserModule } from '@angular/platform-browser';
+    import { NgModule } from '@angular/core';
+    import { AppComponent } from './app.component';
+
+    import { DxFormModule } from 'devextreme-angular';
+
+    @NgModule({
+        declarations: [
+            AppComponent
+        ],
+        imports: [
+            BrowserModule,
+            DxFormModule
+        ],
+        providers: [ ],
+        bootstrap: [AppComponent]
+    })
+    export class AppModule { }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxForm 
+            :form-data="employee"
+            :col-count="2"
+            label-location="top"
+            :show-colon-after-label="false">
+            <DxItem data-field="name"/>
+            <DxItem data-field="position"/>
+            <DxItem data-field="hireDate"/>
+            <DxItem data-field="officeNumber"/>
+            <DxItem 
+                data-field="notes"
+                :col-span="2">
+                <DxLabel alignment="center"/>
+            </DxItem>
+        </DxForm>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxForm, DxItem, DxLabel } from 'devextreme-vue/form';
+    
+    const employee = {
+        name: 'John Heart',
+        position: 'CEO',
+        hireDate: new Date(2012, 4, 13),
+        officeNumber: 901,
+        notes: 'John has been in the Audio/Video industry since 1990.'
+    };
+
+    export default {
+        components: {
+            DxForm,
+            DxItem,
```

Comment:
There are individual components for each item type: `DxSimpleItem`, `DxGroupItem`, etc. Use them instead.